package.json

```
@@ -22,7 +22,12 @@
     "todo": "leasot '**/*.ts' --ignore 'node_modules/**/*','lib/**/*','.git/**/*' || true",
     "watch": "tsc --watch",
     "clean": "rimraf build/",
-    "build": "tsc"
+    "build": "tsc",
+    "dev:load:commonjs": "export RUNTIME_ENVIRONMENT=development; yarn build && time node -e 'const StrongConfig = require(\".\"); console.log(\"Example Config:\", (new StrongConfig()).load(\"./example\"))'",
+    "dev:load:es6": "export RUNTIME_ENVIRONMENT=development; time ts-node -e 'import StrongConfig from \".\"; console.log(\"Example Config:\", (new StrongConfig()).load(\"example\"))'",
```

same formatting suggestion here, and also i _think_ we can shorten the bash command a tiny bit here (you can define env vars in front of any bash command that are valid for this specific command):

```suggestion
    "dev:load:es6": "RUNTIME_ENVIRONMENT=development time ts-node -e 'import StrongConfig from \".\"; console.log(\"Example Config:\\n\", (new StrongConfig()).load(\"example\"))'",
```

---

Do you know if that works on Windows+Linux? Any ways to tests this without having access to a Windows/Linux machine?

---

For Linux you can just ssh into any linux-based docker container on your local machine

windows: good point, to be honest i don't know about syntax there. i usually test using one of their free VMs: https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/

(btw this is something where gitlab actions will be nice as they offer different OS runners for testing)

---

used `cross-env` now